src/test/java/org/apache/sling/distribution/queue/impl/simple/QueueItemMapperTest.java

```diff
@@ -22,13 +22,15 @@
 import java.util.Map;
 
 import org.apache.sling.distribution.queue.DistributionQueueItem;
+import org.junit.Ignore;
 import org.junit.Test;
 
 import static org.junit.Assert.*;
 
 public class QueueItemMapperTest {
 
     @Test
+    @Ignore
```

Comment:
This should not be ignored.

Reply:
The current master branch wasn't running properly when I compiled it. But sure, I will investigate.

Reply:
Ack. Perhaps @cschneider / @tmaret have a bette idea.

Reply:
The problem here is a [local dependent parsing of numbers in QueueMapperImpl](https://github.com/apache/sling-org-apache-sling-distribution-core/blob/master/src/main/java/org/apache/sling/distribution/queue/impl/simple/QueueItemMapper.java#L55). We can defintely hardocde it to English locale (and have the "." as decimal separator) to fix it.

Reply:
I hardcoded the locale now to English. As we are only parsing numbers, I don't consider the impact too big. Nevertheless it's an issue in the existing code and this should be considered a changed behavior.